src/stats/skills.js

```diff
@@ -12,10 +12,7 @@ async function getLevels(userProfile, profileMembers, hypixelProfile, levelCaps)
     }, 0);
 
     Object.assign(skillLevels, {
-      taming: getLevelByXp(SKILL.SKILL_TAMING, {
-        skill: "taming",
-        cap: Math.max(hypixelProfile.achievements.skyblock_domesticator, 50),
-      }),
+      taming: getLevelByXp(SKILL.SKILL_TAMING, {skill: "taming", cap: levelCaps.taming }),
```

Comment:
raising taming cap requires submitting pets to george, so this must be determined by achievement level

Reply:
The problem is that this don't work, since the update where you can give the pets to George in a random order.

Reply:
The Domesticator achievement only give information about the 5/10/15/20 and 25 taming levels, nothing to do whit George (https://wiki.hypixel.net/Achievements)

Reply:
yeah but it's still a lot better than no cap, now it won't be accurate at all, imo this PR shouldn't be merged

Reply:
> @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.

never mind farming is working fine but taming is not, why not set cap 50 directly then?

Reply:
> @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.

what are you talking about? farming works?

Reply:
> > @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.
> 
> never mind farming is working fine but taming is not, why not set cap 50 directly then?

cuz using the achievement is better than settings it to just 50 or 60, and also we at SkyHelper do it that way as well so we should stay consistent in my opinion

Reply:
> > @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.
> 
> never mind farming is working fine but taming is not, why not set cap 50 directly then?

cuz using the achievement is better than settings it to just 50 or 60, and also we at SkyHelper do it that way as well so we should stay consistent in my opinion

Reply:
Ok I will make the change back but the Max level for taming need to be 60 or else you will have a gold bar for not a absolute max level